tests/syntax_tests/data/printer/expr/expected/jsx.res.txt

```diff
@@ -51,7 +51,7 @@ let x =
 let x = <div className="container" className2="container2" className3="container3" onClick />
 
 let nav =
-  <Nav isMobile fullScreen={!isMobile ? false : isOpen}>
+  <Nav isMobile={isMobile} fullScreen={!isMobile ? false : isOpen}>
```

Comment:
Restored this behaviour where `isMobile={isMobile}` in the source is not reformatted to the punned `isMobile`.
Is this intended?

Reply:
IIRC we don't autopun in JSX, yes.

Reply:
It's not punned because of the brackets.

This would be punned
`<Nav isMobile=isMobile fullScreen={!isMobile ? false : isOpen}>`

Reply:
Right, that's it yeah. I think that's good behavior.